web/app/scripts/src/services/fields-config.es.js

```diff
@@ -6,6 +6,7 @@ export const moduleProperties = {
 };
 
 const module = angular.module(moduleProperties.moduleName, [
+    'incuna-surveys-fields.templates',
```

Comment:
@wytrych without this it raises a template not found error. The test site should probably be using the minified templates rather than ajax loading them.